### DISCUSSION TITLE
Passing variable metadata filter in Pinecone Load Existing Index

### DISCUSSION THREAD

**Discussion by muratlutfigoncu** [FlowiseAI/Flowise#1187] [Q&A]
Hey folks, is it possible somehow to dynamically pass metadata filter as a variable while using Pinecone load existing index ?
What I'm trying to do is chaining could of prompt and let LLM to figure out the next action to take. And with the predicted actions and options from the LLM, want to filter Pinecone index

![Screenshot 2023-11-07 104435](https://github.com/FlowiseAI/Flowise/assets/15037968/25d508e0-2f8f-49ba-8ce7-15e8b466dac4)

Thanks!

**Comment by HenryHengZJ** [comment]
You can do so via API `overrideConfig` and specify the `pineconeMetadataFilter`:

![image](https://github.com/FlowiseAI/Flowise/assets/26460777/8f9c2882-e818-4282-aba9-c6fb28c83065)

**Comment by harshZoft** [comment]
Hi I want to set a string variable botId and pass it as a metadata filter on pinecone parameters. But this method doesn't seem to work for me. After creating variable and setting its value, I am setting the filter value as {{$vars.botId}}
